### DISCUSSION TITLE
Switching away from Slack?

### DISCUSSION THREAD

**Discussion by paulcretu** [orcasound/.github#16] [Ideas]
Problems with Slack:

- On the free tier, you can only search back 90 days of messages. This makes it hard to maintain institutional knowledge and to rely on past discussions
- Limited to 10 integrations on free tier
- Paid tier is prohibitively expensive (even with 85% non-profit discount it would still be expensive):
<img width="389" alt="image" src="https://github.com/orcasound/.github/assets/1069628/b370447d-8d3d-4c91-82e6-7fbdf205faf5">

- Slack requires an invite to join and view. This doesn't line up with the openness that Orcasound aims for. It would be preferable to having a publicly viewable community that contributors can first browse and then choose to join once they want to post.
- Anecdotally, it's been very noisy lately and hard to keep up with everything going on. For me at least, Slack's notification settings haven't been enough.

Advantages of Slack:
- We're already using it
- Many people are using it for work/other projects so they already have the app installed and running, know how to use it
- Good mobile app
- Very low management overhead (no need to self-host)
- Pretty good integrations with anything you want (e.g. [Google Docs](https://slack.com/apps/A6NL8MJ6Q-google-drive), [Heroku Chat Ops](https://slack.com/apps/A1QME020P-heroku-chatops))

Alternatives to consider:
- [Discord](https://discord.com)
- [Zulip](https://zulip.com)
- [Mattermost](https://mattermost.com)
- [Element/Matrix](https://element.io)
- [Rocket.Chat](https://rocket.chat)

**Comment by scottveirs** [comment]
Here's a web3 Slack alternative that uses OrbitDB and IPFS like Acartia.io !

https://github.com/TryQuiet/quiet

**Comment by scottveirs** [comment]
PSA: We have left Slack for Zulip.

[You're invited to join us](https://orcasound.zulipchat.com/join/42uml7y5yr7n7pbc7k2icbcb/) in the new, more-open Orcasound chat!
